internal/configs/configmaps.go

```diff
@@ -207,11 +207,21 @@ func ParseConfigMap(cfgm *v1.ConfigMap, nginxPlus bool, hasAppProtect bool, hasA
 		cfgParams.MainErrorLogLevel = errorLogLevel
 	}
 
+	if accessLog, exists := cfgm.Data["access-log"]; exists {
+		if !strings.HasPrefix(accessLog, "syslog:") {
```

Comment:
seems to allow stuff other than syslog
```
Name:         test-release-nginx-ingress
Namespace:    default
Labels:       app.kubernetes.io/instance=test-release
              app.kubernetes.io/managed-by=Helm
              app.kubernetes.io/name=nginx-ingress
              app.kubernetes.io/version=3.7.0
              helm.sh/chart=nginx-ingress-1.4.0
Annotations:  meta.helm.sh/release-name: test-release
              meta.helm.sh/release-namespace: default

Data
====
access-log:
----
yslog:server=syslog-svc.default:514

BinaryData
====

Events:
  Type    Reason   Age                From                      Message
  ----    ------   ----               ----                      -------
  Normal  Updated  11s (x5 over 21m)  nginx-ingress-controller  Configuration from default/test-release-nginx-ingress was updated
```

Reply:
OK any idea why?

Reply:
OK it's allowed but ignored (with an error log) which is the case for all other configmap items as far as I understand

Reply:
@hafe we would ideally like it to make configmap invalid

Reply:
looking into this case atm

Reply:
This works as expected (implemented in the code atm).

When user sets the access-log-off to `true` and the value of `access-log` does not meet the condition (starts with `syslog`) the log is set to it's default value: `access_log /dev/stdout main;`